### PR TITLE
Fix key navigation in scene explorer

### PR DESCRIPTION
**Pull request by sebavan**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/inspector/src/components/sceneExplorer/sceneExplorerComponent.tsx [hunk 1]
```diff
@@ -507,6 +507,24 @@ export class SceneExplorerComponent extends React.Component<ISceneExplorerCompon
...
+        allNodes.push(
+            rootNodes,
+            scene.skeletons,
+            materials,
+            textures,
+            postProcesses,
+            pipelines,
+            scene.effectLayers,
+            scene.particleSystems,
+            scene.particleSystems,
```
Comment (bghgary) [inline packages/dev/inspector/src/components/sceneExplorer/sceneExplorerComponent.tsx:519]: Should this be here twice?

**Review by bghgary — APPROVED** [review]

File: packages/dev/inspector/src/components/sceneExplorer/sceneExplorerComponent.tsx (hunk 1 above)
Comment (sebavan) [reply packages/dev/inspector/src/components/sceneExplorer/sceneExplorerComponent.tsx]: nope, thanks

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14450/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14450/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14450/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14450/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14450/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14450/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14450/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14450/merge#BCU1XR#0

**Pull request merged**